tests/integration_test/dynamic_routing_integration_test.go

```diff
@@ -191,6 +191,42 @@ func TestDynamicRouting(t *testing.T) {
 			method:        "GET",
 			httpCallError: fmt.Errorf("http response status is not 200 OK: 404 Not Found"),
 		},
+		{
+			desc:     "Succeed, CONSTANT_ADDRESS with single segments in double wildcards",
+			path:     "/wildcard/a/1/b/2/c/3",
+			method:   "GET",
+			wantResp: `{"RequestURI":"/dynamicrouting/const_wildcard?name=2"}`,
+		},
+		{
+			desc:     "Succeed, CONSTANT_ADDRESS with multiple segments in double wildcards",
+			path:     "/wildcard/a/1/b/2/c/3/4/5",
+			method:   "GET",
+			wantResp: `{"RequestURI":"/dynamicrouting/const_wildcard?name=2"}`,
+		},
+		{
+			desc:     "Succeed, CONSTANT_ADDRESS with no segments in double wildcards",
+			path:     "/wildcard/a/1/b/2/c/",
```

Comment:
What will happen for "/wildcard/a/1/b/2/c"  without last "/"?

Reply:
Good catch, it didn't work. I fixed it.

Reply:
I see, I reverted that change.

Reply:
Still kept the test case to show 404.